secure/json-rpc/src/tests/client.rs

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{tests::mock_libra_db::MockLibraDB, JsonRpcClient, LightweightJsonClient};
+use futures::{channel::mpsc::channel, StreamExt};
+use libra_config::utils;
+use libra_crypto::{ed25519::Ed25519PrivateKey, HashValue, PrivateKey, Uniform};
+use libra_json_rpc::bootstrap;
```

Comment:
Generally, I agree with you. However, in this case, it's seems like someone has already explicitly exported functions from the runtime file in the libra_json_rpc package to avoid making the file public (e.g., see the 'bootstrap_from_config' export in the 'lib.rs' file). If you want to make the entire thing public we can do that, but I wanted to keep the original semantic structure (to avoid issues I may not be aware about...)